core/gui/src/app/workspace/component/menu/menu.component.ts

```diff
@@ -248,6 +249,21 @@ export class MenuComponent implements OnInit {
     this.executeWorkflowService.takeGlobalCheckpoint();
   }
 
+  /**
+   * get the html to export all results.
+   */
+  public onClickExportAllResults(): void {
+    this.reportPrintService
+      .getWorkflowSnapshot()
+      .pipe(
+        tap((snapshot: string) => this.reportPrintService.downloadResultsAsHtml(snapshot)),
+        untilDestroyed(this)
+      )
+      .subscribe({
+        error: (error: unknown) => console.error("Error capturing workflow snapshot:", (error as Error).message),
```

Comment:
use notificationService, a existing common component, to log this message out. For how to use `notificationService`, you can check other classes.